addons/website/views/snippets.xml

```diff
@@ -1466,6 +1466,20 @@
         <we-checkbox string="Stretch" data-select-class="container|container-fluid" data-no-preview="true"/>
     </div>
 
+    <!-- Scroll to next section button -->
+    <div data-js="ScrollButton"
+        data-selector="section">
+        <we-row>
+            <we-title>Scroll down button</we-title>
+            <we-colorpicker class="ml-auto"
+                data-select-style="true"
+                data-css-property="color"
+                data-color-prefix="text-"
+                data-apply-to=".o_scroll_button"/>
```

Comment:
Should not be visible if no button... shit I think I forgot apply-to with the visibility stuff :/